server/server.go

```diff
@@ -243,15 +247,26 @@ func New(cfg Config) (*Server, error) {
 		grpcListener = netutil.LimitListener(grpcListener, cfg.GRPCConnLimit)
 	}
 
+	cipherSuites, err := stringToCipherSuites(cfg.CipherSuites)
+	if err != nil {
+		return nil, err
+	}
+	minVersion, err := stringToTLSVersion(cfg.MinVersion)
+	if err != nil {
+		return nil, err
+	}
+
 	// Setup TLS
 	var httpTLSConfig *tls.Config
 	if len(cfg.HTTPTLSConfig.TLSCertPath) > 0 && len(cfg.HTTPTLSConfig.TLSKeyPath) > 0 {
 		// Note: ConfigToTLSConfig from prometheus/exporter-toolkit is awaiting security review.
 		httpTLSConfig, err = web.ConfigToTLSConfig(&web.TLSStruct{
-			TLSCertPath: cfg.HTTPTLSConfig.TLSCertPath,
-			TLSKeyPath:  cfg.HTTPTLSConfig.TLSKeyPath,
-			ClientAuth:  cfg.HTTPTLSConfig.ClientAuth,
-			ClientCAs:   cfg.HTTPTLSConfig.ClientCAs,
+			TLSCertPath:  cfg.HTTPTLSConfig.TLSCertPath,
+			TLSKeyPath:   cfg.HTTPTLSConfig.TLSKeyPath,
+			ClientAuth:   cfg.HTTPTLSConfig.ClientAuth,
+			ClientCAs:    cfg.HTTPTLSConfig.ClientCAs,
+			CipherSuites: cipherSuites,
+			MinVersion:   minVersion,
```

Comment:
Shouldn't these also be added to `grpcTLSConfig`?

It sounds from the PR description as if they're supposed to be for both HTTP and gRPC:

> Add a single parameter for each, not split across HTTP and gRPC.